### ISSUE TITLE
Usage of fprintf instead of GREATEST_FPRINTF

### ISSUE THREAD

**Issue by Nicolapps** [silentbicycle/greatest#117]
On lines 761 and 1119 of `greatest.h`, `fprintf` is used. Shouldn’t `GREATEST_FPRINTF` be used instead, to allow the usage of the library in cases where `fprintf` isn’t available and the user would like to use another function instead?

https://github.com/silentbicycle/greatest/blob/11a6af1919049df502405913da64fb385c219361/greatest.h#L761
https://github.com/silentbicycle/greatest/blob/11a6af1919049df502405913da64fb385c219361/greatest.h#L1119

**Comment by silentbicycle** [comment]
Good catch, thanks. These will be fixed in the next release, and I'll check if any others got missed.

**Comment by Nicolapps** [comment]
Thank you for your reply!
